tests/console/systemd_testsuite.pm

```diff
@@ -0,0 +1,53 @@
+# SUSE's systemd tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Run testsuite included in systemd sources
+# Maintainer: Thomas Blume <tblume@suse.com>
+
+use base "consoletest";
+use warnings;
+use strict;
+use testapi;
+use utils;
+
+sub run() {
+    # install systemd testsuite
+    select_console 'root-console';
+    if (check_var('VERSION', '12-SP2')) {
```

Comment:
There is already a test package at:

https://build.suse.de/package/show/QA:Head/qa_test_systemd

but these tests are unrelated to the systemd testsuite.
Do you want me to merge the systemd-testsuite tests there or to create a new package?